lib/Basic/FileInfo.cpp

```diff
@@ -46,6 +48,9 @@ FileInfo FileInfo::getInfoForPath(const std::string& path, bool asLink) {
 #if defined(__APPLE__)
   auto seconds = buf.st_mtimespec.tv_sec;
   auto nanoseconds = buf.st_mtimespec.tv_nsec;
+#elif defined(_WIN32)
+  auto seconds = buf.st_mtime;
+  auto nanoseconds = 0;
```

Comment:
Doesn't seem like it, at least not with `_stat`. I'll take a look for other APIs, though